### DISCUSSION TITLE
Page rotation

### DISCUSSION THREAD

**Discussion by OtakuHK** [R0Wi-DEV/workflow_ocr#205] [Ideas]
Would it be possible to have an option to activate automatic page rotation. As far as i can see OCRmyPDF supports this with --rotate-pages command.

Kind regards

**Comment by R0Wi** [comment]
Sure that's possible. Ref: https://ocrmypdf.readthedocs.io/en/latest/cookbook.html#correct-page-rotation

I'd suggest to adjust the workflow settings UI to contain a additional checkbox "Rotate pages". Default would be `false`.

**Reply by OtakuHK** [reply]
That would be awesome!
